src/main/java/fi/aalto/cs/apluscourses/intellij/actions/ModuleFilterOptionAction.java

```diff
@@ -0,0 +1,30 @@
+package fi.aalto.cs.apluscourses.intellij.actions;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.project.DumbAwareToggleAction;
+import fi.aalto.cs.apluscourses.presentation.filter.Option;
+import org.jetbrains.annotations.NotNull;
+
+public class ModuleFilterOptionAction extends DumbAwareToggleAction {
```

Comment:
Is it reasonable have a separate class instead of reusing the one of the exercise filters?